release-notes/1.0/RC2_Contributor_List.md

```diff
@@ -0,0 +1,404 @@
+* [Rama (1)](https://github.com/dotnet/coreclr/commits/master?author=ramarag@microsoft.com)
+* [DionYe (1)](https://github.com/dotnet/coreclr/commits/master?author=junye@microsoft.com)
+* [Eugene Zemtsov (1)](https://github.com/dotnet/coreclr/commits/master?author=e.zemtsov@gmail.com)
+* [Ludovic Henry (1)](https://github.com/dotnet/coreclr/commits/master?author=ludovic@xamarin.com)
+* [Felipe Pessoto (1)](https://github.com/dotnet/coreclr/commits/master?author=felipepessoto@hotmail.com)
+* [Lakshmi Priya (1)](https://github.com/dotnet/coreclr/commits/master?author=Priya91@users.noreply.github.com)
+* [noahfalk (1)](https://github.com/dotnet/coreclr/commits/master?author=noahfalk@users.noreply.github.com)
+* [Ron Cain (1)](https://github.com/dotnet/coreclr/commits/master?author=roncain@microsoft.com)
+* [Chris (1)](https://github.com/dotnet/coreclr/commits/master?author=Chrisboh@microsoft.com)
+
+## CoreFX
+
+* [Stephen Toub (798)](https://github.com/dotnet/corefx/commits/master?author=stoub@microsoft.com)
+* [stephentoub (356)](https://github.com/dotnet/corefx/commits/master?author=stoub@microsoft.com)
+* [Eric Mellino (138)](https://github.com/dotnet/corefx/commits/master?author=erme@microsoft.com)
+* [Eric St. John (113)](https://github.com/dotnet/corefx/commits/master?author=ericstj@microsoft.com)
+* [Hugh Bellamy (112)](https://github.com/dotnet/corefx/commits/master?author=hughbellars@gmail.com)
+* [David Shulman (109)](https://github.com/dotnet/corefx/commits/master?author=david.shulman@microsoft.com)
+* [Ian Hays (93)](https://github.com/dotnet/corefx/commits/master?author=ianha@microsoft.com)
+* [Jon Hanna (90)](https://github.com/dotnet/corefx/commits/master?author=jon@hackcraft.net)
+* [Eric Erhardt (85)](https://github.com/dotnet/corefx/commits/master?author=eric.erhardt@microsoft.com)
+* [bartonjs (79)](https://github.com/dotnet/corefx/commits/master?author=jbarton@microsoft.com)
+* [Cory Rivera (77)](https://github.com/dotnet/corefx/commits/master?author=corivera@microsoft.com)
+* [Matt Ellis (70)](https://github.com/dotnet/corefx/commits/master?author=matell@microsoft.com)
+* [Wes Haggard (61)](https://github.com/dotnet/corefx/commits/master?author=weshaggard@users.noreply.github.com)
+* [VSadov (55)](https://github.com/dotnet/corefx/commits/master?author=vsadov@microsoft.com)
+* [Wes Haggard (53)](https://github.com/dotnet/corefx/commits/master?author=Wes.Haggard@microsoft.com)
+* [Eric Eilebrecht (51)](https://github.com/dotnet/corefx/commits/master?author=ericeil@hotmail.com)
+* [Lakshmi Priya Sekar (50)](https://github.com/dotnet/corefx/commits/master?author=lasekar@microsoft.com)
+* [Matt Mitchell (48)](https://github.com/dotnet/corefx/commits/master?author=mmitche@microsoft.com)
+* [Eric StJohn (47)](https://github.com/dotnet/corefx/commits/master?author=ericstj@microsoft.com)
+* [Saurabh Singh (43)](https://github.com/dotnet/corefx/commits/master?author=saurabh500@gmail.com)
```

Comment:
@saurabh500 - looks like mapping to the username would required going in through the git APIs rather than querying the log as I am now. Perhaps something I'll try in the future.